cypress/e2e/SiteAlert/StandardAlert.feature

```diff
@@ -0,0 +1,159 @@
+            | prop8     | english                                                                    |
+            | evar2     | D=c8                                                                       |
+            | prop52    | D=v52                                                                      |
+            | prop68    | D=v68                                                                      |
+            | pageURL   | https://{CANONICAL_HOST}/                                                  |
+            | pageName  | {CANONICAL_HOST}/                                                          |
+            | event83   |                                                                            |
+            | channel   | NCI Homepage                                                               |
+            | pev2      | PreHeader:LinkClick                                                        |
+            | linkType  | lnk_o                                                                      |
+            | link      | What people with cancer should know                                        |
+
+    Scenario: Analytics for site alert dismiss click
+        Given user is navigating to "/"
+        Then standard site alert is displayed
+        When user dismisses alert
+        Then page click request is sent
+        And the following parameters should be captured
+            | parameter | value                                       |
+            | prop4     | D=pev1                                      |
+            | prop67    | D=pageName                                  |
+            | evar52    | PreHeader\|Standard Alert\|Dismiss\|Dismiss |
+            | evar68    | PreHeader                                   |
+            | prop8     | english                                     |
+            | evar2     | D=c8                                        |
+            | prop52    | D=v52                                       |
+            | prop68    | D=v68                                       |
+            | pageURL   | https://{CANONICAL_HOST}/                   |
+            | pageName  | {CANONICAL_HOST}/                           |
+            | event83   |                                             |
+            | channel   | NCI Homepage                                |
+            | pev2      | PreHeader:LinkClick                         |
+            | linkType  | lnk_o                                       |
+        When user is navigating to "/about-cancer/coping/feelings"
+        Then standard site alert is not displayed
+
+    Scenario: login to remove alert (so after hook doesn't have to)
+        Given user is navigating to "/user/login"
+        When user enters credentials
+        And user clicks "Log in" button
+        When user is logged in and the user name "admin" is displayed in the toolbar
```

Comment:
Does this seem to end abruptly, i.e. it is not deleted?

Reply:
this is just logging in for the after hook, so I don't have to duplicate the login code in a hook)